rwandaemr-imb-butaro/spa-build-config.json

```diff
@@ -1,7 +1,7 @@
 {
   "frontendModules":{
-    "@pih/esm-pathology-app":"${pathologyappVersion}",
-    "@pih/esm-legacyui-navbar-app":"${legacyuinavbarappVersion}"
+    "@pih/esm-pathology-app":"1.1.1-pre.29",
+    "@pih/esm-legacyui-navbar-app":"1.0.1-pre.1"
```

Comment:
Just putting the versions directly in here, as I was having difficulty with the variables.